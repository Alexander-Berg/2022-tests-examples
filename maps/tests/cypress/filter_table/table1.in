{"id"="0";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="1";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b2";"C"="c2";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="2";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid1"};
{"id"="3";"edge_id"=0u;"segment_index"=0u;"A"="b";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="4";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b2";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="5";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20150101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="6";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T001500";"clid"="clid0";"uuid"="uuid0"};
{"id"="7";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=38.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="8";"edge_id"=0u;"segment_index"=1u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="9";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160102T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="10";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=1u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
