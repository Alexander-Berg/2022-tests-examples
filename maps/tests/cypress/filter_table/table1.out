{"id"="0";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b1";"C"="c1";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
{"id"="1";"edge_id"=0u;"segment_index"=0u;"A"="a";"B"="b2";"C"="c2";"region_id"=213u;"lat"=55.7746;"lon"=37.6513;"time"="20160101T000500";"clid"="clid0";"uuid"="uuid0"};
